src/tests/playwright/pages/chuck-realtime-sine.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script defer type="module">
+      import {record} from './util/recorder/recorder-main.js';
+      import {beCloseTo, test, evaluateTest} from './util/audit.js';
+      import {Chuck} from 'https://cdn.jsdelivr.net/npm/webchuck/+esm';
+
+      const expectedAccuracy = 0.998;
+
+      test((async () => {
+        const context = new AudioContext({sampleRate: 48000});
+        await context.suspend();
+        // Initialize default ChucK object
+        const theChuck = await Chuck.init([], context, 1);
+        const duration = 1; // second
+        const {recorder, recordingCompletePromise} = await record(context, duration);
+        theChuck.connect(recorder).connect(context.destination);
+
+        // Run ChucK code
+        theChuck.runCode(`
+          SinOsc sin => dac;
+          440 => sin.freq;
+          1::second => now;
+        `);
+        await context.resume();
+
+        const recordedChannelBuffers = await recordingCompletePromise;
+        const float32Buffer = recordedChannelBuffers[0];
+
+        await context.close();
+        return {buffer: float32Buffer};
+      })());
+
+      evaluateTest(async (testResult) => {
+        const bufferData = testResult.buffer;
+        const myRefData = await (await fetch('./reference/440@48k-sine-octx.json')).json();
+
+        // compare bufferData samples to reference
+        let numberOfAcceptableSamples = 0;
+        for (let i = 0; i < bufferData.length; i++) {
```

Comment:
Perhaps it's better to add this routine as a helper function to `audit.js`?

It looks like we are comparing the buffer data with the json values.